src/components/pages/AdminDashboard/DevTools/DayComponents/01_Sat-Mon.js

```diff
@@ -41,6 +38,28 @@ const SatMon = ({ setDate }) => {
     setDate(findNextDayOfWeek(findDayOfWeekReference));
   }, [findDayOfWeekReference]);
 
+  const onChange = e => {
+    setValue(e.target.value);
+    console.log('tasks: ', radioTasks);
+  };
+
+  const handleGetChildTasks = async e => {
+    const res = await getChildTasks(authState, child.id, child.cohortId);
```

Comment:
we should wrap all of the logic in handleGetChildTasks in a `try{}catch(err){}` in case one of the API calls fail.